tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNet/ReflectedHttpActionDescriptor_ExecuteAsync_Integration.cs

```diff
@@ -83,11 +84,18 @@ public class ReflectedHttpActionDescriptor_ExecuteAsync_Integration
                     {
                         var context = HttpContext.Current;
                         var scope = SharedItems.TryPeekScope(context, AspNetWebApi2Integration.HttpContextKey);
```

Comment:
IMO we need to add `#nullable enable` to this file, as well as the `SharedItems.TryPeekScope` one, as there are several other potential issues of the same kind that I've seen